src/celeritas/phys/PhysicsParams.hh

```diff
@@ -60,7 +56,6 @@ struct PhysicsParamsOptions
     real_type fixed_step_limiter     = 0;
     real_type linear_loss_limit      = 0.01;
     real_type secondary_stack_factor = 3;
-    bool      use_integral_xs        = true;
```

Comment:
Maybe it would be better to hardcode "supports integral xs" into each process (since it doesn't depend on the data and doesn't make sense for a user to toggle on a per-process basis), and have the global physics option be "disable integral xs" ?

Reply:
Geant4 does let you set this on a per-process basis; from the PRM:

> The integral variant of step limitation is the default for the G4eIonisation, G4hIonisation, G4eBremsstrahlung and some other processes for charged particles.

Electron-positron annihilation also has it on by default, but others like muon bremsstrahlung and muon ionization don't. So maybe it makes sense to keep some flexibility here?

Reply:
Agreed, let's keep the flexibility.